src/pages/patch/ScheduleUndispatchedBaseTasks.tsx

```diff
@@ -0,0 +1,50 @@
+import { useMutation } from "@apollo/client";
+import { Body } from "@leafygreen-ui/typography";
+import { Popconfirm } from "antd";
+import { DropdownItem } from "components/ButtonDropdown";
+import { useToastContext } from "context/toast";
+import {
+  ScheduleUndispatchedBaseTasksMutation,
+  ScheduleUndispatchedBaseTasksMutationVariables,
+} from "gql/generated/types";
+import { SCHEDULE_UNDISPATCHED_BASE_TASKS } from "gql/mutations";
+
+interface Props {
+  patchId: string;
+}
+export const ScheduleUndispatchedBaseTasks: React.FC<Props> = ({ patchId }) => {
+  const dispatchToast = useToastContext();
+  const [scheduleBasePatchTasks] = useMutation<
+    ScheduleUndispatchedBaseTasksMutation,
+    ScheduleUndispatchedBaseTasksMutationVariables
+  >(SCHEDULE_UNDISPATCHED_BASE_TASKS, {
```

Comment:
when this finishes, does it also update the base tasks' statuses on this page or does that require a refresh?

Reply:
It should, the response from the mutation should automatically update the tasks in the apollo cache and then update the page